api/Controllers/GameController.cs

```diff
@@ -32,5 +33,15 @@ public async Task<IActionResult> GetSingle(string id)
             var result = await _dbService.GetSingleGame(id);
             return Ok(result);
         }
+
+        [HttpPost("new")]
```

Comment:
I'd recommend we just change this to a HttpPost without any additional path to better follow rest api conventions.

Reply:
Updated